capstone-backend/src/test/java/com/google/sps/data/BookListTest.java

```diff
@@ -52,27 +50,31 @@ public void testContructors() {
     assertNotEquals(bookList1, bookList2);
 
     // Test Diff
-    assertNotEquals(bookList1.getUserID(), bookList2.getUserID());
-    assertNotEquals(bookList1.getBookshelfID(), bookList2.getBookshelfID());
+    assertEquals(false, bookList1.getUserID().equals(bookList2.getId()));
 
-    assertNotEquals(bookList1.getBookshelfID(), bookList2.getBookshelfID());
+    assertEquals(false, bookList1.getBookshelfID().equals(bookList2.getBookshelfID()));
 
-    assertNotNull(bookList1.getId());
-    assertNull(bookList2.getId());
+    assertNotEquals(bookList1.getCollaboratorsIDs(), bookList2.getCollaboratorsIDs());
+
+    assertEquals(false, bookList1.getCollaboratorsIDs().containsAll(bookList2.getCollaboratorsIDs())
+        && bookList2.getCollaboratorsIDs().containsAll(bookList1.getCollaboratorsIDs()));
+
+    assertEquals(false, bookList1.getId().equals(bookList2.getId()));
```

Comment:
Read comments from below - it's not best practice to have `assertEquals(false..`

Reply:
Got it